javascript/sbp/ssr.js

```diff
@@ -29,6 +29,7 @@ var CarrierPhase = require("./gnss").CarrierPhase;
 var GPSTime = require("./gnss").GPSTime;
 var GPSTimeSec = require("./gnss").GPSTimeSec;
 var GPSTimeDep = require("./gnss").GPSTimeDep;
+var None = require("./observation").None;
```

Comment:
@mfine the generator seems to mess up the import of ObservationHeader, any idea..?